src/app/http/http.service.ts

```diff
@@ -18,7 +18,8 @@ export class HttpService {
 }
 // tslint:disable-next-line: class-name
 export interface newMovie {
```

Comment:
nie sądzę żeby new było potrzebne w tej nazwie, ale jak już to interfejsy nazwyamy wielką literą